src/drv_vrtek/vrtek.c

```diff
@@ -0,0 +1,548 @@
+// Copyright 2019, Mark Nelson.
+// SPDX-License-Identifier: BSL-1.0
+/*
+ * OpenHMD - Free and Open Source API and drivers for immersive technology.
+ */
+
+/* VR-Tek Driver */
+
+#include <stdlib.h>
+#include <hidapi.h>
+#include <string.h>
+#include <stdio.h>
+#include <assert.h>
+
+#include "vrtek.h"
+#include "../hid.h"
+
+#define VRTEK_ID       0x2833
```

Comment:
Let's not lie, this is OCULUS_VR_INC_ID, not VRTEK_ID.

Reply:
Done. I just pushed a commit that updated the name of that macro and added a comment explaining that Oculus' vendor ID is being reused here.